libandroid-navigation-ui/src/main/res/values/dimens.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <dimen name="fab_margin_bottom">8dp</dimen>
```

Comment:
Can we adjust this to `16dp`?  On higher res devices, the buttons are too close imo, may be more easy to click the wrong one:

![device-2018-04-24-083032](https://user-images.githubusercontent.com/8434572/39187058-3af6d0b4-479a-11e8-9beb-e62881375e55.png)

Reply:
Nah, that's not necessary 👀 `fab_margin_bottom` which doesn't affect the distance between sound and feedback buttons (it's the same exact value as it was).